packages/ui-signer/src/Extrinsic.tsx

```diff
@@ -40,10 +57,23 @@ class Transaction extends React.PureComponent<Props> {
           <div className='expanded'>
             <p>
               <Trans i18nKey='decoded.short'>
-                You are about to sign a message from <span className='code'>{accountId}</span> calling <span className='code'>{section}.{method}</span> with an index of <span className='code'>{accountNonce.toString()}</span>
+                You are about to sign a message from <span className='code'>{accountId}</span> calling <span className='code'>{section}.{method}</span> with an index of <span className='code'>{nonce.toString()}</span>
               </Trans>
             </p>
-            <Call value={extrinsic} />
+            <Call value={extrinsic}/>
+            <div
+              className={'ui--signer-Signer-FeeDisplay'}
+              style={style}
+            >
+              <FeeDisplay
```

Comment:
This is not quite correct - but it does work... So the issue is that this component takes the `transferFee` into account as well, but it is only applicable for, well, balances.transfer. It does work at this point since it is set to `0` on-chain, so the match work. However as soon as that value changes, the values for non-transfers will be incorrect.